packages/thumbprint-react/components/Popover/index.jsx

```diff
@@ -0,0 +1,219 @@
+     */
+    children: PropTypes.node,
+    /**
+     * A function that renders JSX and receives an object with `ref`.
+     * All of these props must be added to the component within the render prop.
+     */
+    launcher: PropTypes.func.isRequired,
+    /**
+     * Position of popover relative to the launcher.
+     */
+    position: PropTypes.oneOf([
+        'top-start',
+        'top',
+        'top-end',
+        'bottom-start',
+        'bottom',
+        'bottom-end',
+        'left-start',
+        'left',
+        'left-end',
+        'right-start',
+        'right',
+        'right-end',
+    ]),
+    /**
+     * Whether or not the popover is visible.
+     */
+    isOpen: PropTypes.bool.isRequired,
+    /**
+     * Function called when the close button is clicked. You should cause this to set `isOpen=false`
+     * in your parent component
+     */
+    onCloseClick: PropTypes.func.isRequired,
+    /**
+     * By default popovers will render right before the `</body>` tag.
+     * Setting the `container` to `inline` causes the tooltip to remain where it was added to the
+     * DOM.
+     * This option is helpful to work around z-index and positioning issues.
+     */
+    container: PropTypes.oneOf(['inline', 'body']),
+};
+
+Popover.defaultProps = {
+    children: null,
+    position: 'top',
+    container: 'body',
+};
+
+const clickOutsideConfig = {
+    handleClickOutside: () => Popover.handleClickOutside,
+};
+
+export default onClickOutside(Popover, clickOutsideConfig);
+
+const PopoverTitle = ({ children }) => <div className={styles.popoverTitle}>{children}</div>;
+
+PopoverTitle.propTypes = {
+    /**
+     * A title for the popover.
+     */
+    children: PropTypes.string.isRequired,
+};
+
+const PopoverBody = ({ children }) => <div className={styles.popoverBody}>{children}</div>;
+
+PopoverBody.propTypes = {
+    /**
+     * The body description text for the popover.
+     */
+    children: PropTypes.string.isRequired,
+};
+
+const PopoverPrimaryButton = ({ children, onClick }) => (
+    <div className={styles.popoverButton}>
+        <Themed size="small" onClick={onClick} theme="popover-primary">
+            {children}
+        </Themed>
+    </div>
+);
+
+PopoverPrimaryButton.propTypes = {
+    /**
+     * The text for this button.
+     */
+    children: PropTypes.string.isRequired,
+    /**
+     * Function called when the button is clicked.
+     */
+    onClick: PropTypes.func.isRequired,
+};
+
+const PopoverSecondaryButton = ({ children, onClick }) => (
+    <div className={styles.popoverButton}>
+        <Themed size="small" onClick={onClick} theme="popover-secondary">
+            {children}
+        </Themed>
+    </div>
+);
+
+PopoverSecondaryButton.propTypes = {
+    /**
+     * The text for this button.
+     */
+    children: PropTypes.string.isRequired,
+    /**
+     * Function called when the button is clicked.
+     */
+    onClick: PropTypes.func.isRequired,
```

Comment:
This may sometimes be a link to another page. (Image a "Learn more" CTA.)

Could use a `to` prop like: https://thumbprint.design/components/link/react/#section-themed-links

Reply:
Done